### PR TITLE
Update hide-application-from-user-portal.md

### PR DESCRIPTION
**Pull request by secherka**
Per conversation with PM and PG agreed to make this clarification to the document

### CODE REVIEW

**Review by secherka — COMMENTED** [review]
removed en-us

File: docs/identity/enterprise-apps/hide-application-from-user-portal.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Use the following steps to hide an application from My Apps portal and Microsoft
 :::zone-end
 
 > [!NOTE]
-> These instructions apply only to Enterprise applications.
+> These instructions apply only to non first-party Microsoft Enterprise Applications. To learn more about first party Microsoft applications see this [link](https://learn.microsoft.com/troubleshoot/azure/entra/entra-id/governance/verify-first-party-apps-sign-in). Administrators also need to keep in mind that hiding the application from the end user does not prevent users signing in these applications via other then My Apps portal means (shared links, services dependencies, etc.).
```
Comment (omondiatieno) [inline docs/identity/enterprise-apps/hide-application-from-user-portal.md:48]:
```suggestion
> These instructions apply only to non first-party Microsoft Enterprise Applications. To learn more about first-party Microsoft applications see [First-party Microsoft applications in sign-in reports](/troubleshoot/azure/entra/entra-id/governance/verify-first-party-apps-sign-in). Administrators also need to keep in mind that hiding the application from the end user does not prevent users signing in these applications via other then My Apps portal means (shared links, services dependencies, etc). 
```

**Review by secherka — COMMENTED** [review]
wording improvement

**Review by secherka — COMMENTED** [review]
additional wording improvement

**Review by secherka — COMMENTED** [review]
additional commit to fix the build failed error

**Comment by omondiatieno** [comment]
#sign-off

File: docs/identity/enterprise-apps/hide-application-from-user-portal.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Use the following steps to hide an application from My Apps portal and Microsoft
 :::zone-end
 
 > [!NOTE]
-> These instructions apply only to Enterprise applications.
+> These instructions apply only to non first-party Microsoft Enterprise Applications. To learn more about first-party Microsoft applications see [First-party Microsoft applications in sign-in reports](/troubleshoot/azure/entra/entra-id/governance/verify-first-party-apps-sign-in). Administrators also need to keep in mind, that hiding the application from the end user does not prevent users signing in these applications via other than My Apps portal methods, such as shared links or service dependencies.
```
Comment (Court72) [inline docs/identity/enterprise-apps/hide-application-from-user-portal.md:48]:
```suggestion
> These instructions apply only to non-first-party Microsoft Enterprise Applications. To learn more about first-party Microsoft applications see [First-party Microsoft applications in sign-in reports](/troubleshoot/azure/entra/entra-id/governance/verify-first-party-apps-sign-in). Administrators also need to keep in mind that hiding the application from the users doesn't prevent them from signing into these applications via methods other than the My Apps portal, such as shared links or service dependencies. 
```

**Pull request merged**